specification/cognitiveservices/OpenAI.Inference/models/completions.create.cadl

```diff
@@ -3,19 +3,10 @@ import "@cadl-lang/rest";
 using Cadl.Rest;
 using Cadl.Http;
 
-namespace Azure.OpenAI;
-
-@doc("Input model for completions create endpoint")
-model CompletionsCreateRequest {
-    @doc("deployment id of the model which was deployed")
-    @path deploymentId: string;
-    @doc("expected schema for the body of the completion post request")
-    @body
-    body: CompletionsCreateBody;
-}
+namespace Azure.OpenAI.Completions;
 
-@doc("Post body schema to create a prompt completion from a deployment")
-model CompletionsCreateBody {
+//@doc("Post body schema to create a prompt completion from a deployment")
```

Comment:
Should we uncomment this `@doc`?

Reply:
Currently you can't decorate an alias, which is why it is commented out.